jenkins-jobs/cloudwatch-metrics/Jenkinsfile

```diff
@@ -0,0 +1,25 @@
+#!groovy
+
+pipeline {
+  agent any
+  triggers {
+    cron('H/5 * * * *')
+  }
+  tools {
+      maven 'Maven 3.5.0'
+      jdk 'jdk8'
+  }
+  stages {
+    stage('Build and Publish Metric') {
+      steps {
+        sh 'mvn clean'
+        sh 'mvn -Dmaven.test.failure.ignore=true package'
+      }
+      post {
+        success {
+          sh 'java -jar target/cloudwatch-metrics-1.0-jar-with-dependencies.jar'
```

Comment:
Where does this JAR come from?

Reply:
It gets built from the java code (`Heartbeat.java`) with Maven (`mvn`) in the steps before this using the `package` command.